code/game/objects/items/cigs_lighters.dm

```diff
@@ -316,7 +325,10 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 
 /obj/item/clothing/mask/cigarette/syndicate
 	desc = "An unknown brand cigarette."
-	list_reagents = list(/datum/reagent/drug/nicotine = 15, /datum/reagent/medicine/omnizine = 15)
+	chem_volume = 60
+	smoketime = 60

```

Comment:
As a side effect, syndi cigs are now the second best cigar bomb, only matched by havanas and bested by cohibas.

Reply:
???

Pipes and ecigs have a reagent capacity of 100u, so I don't know what you're talking about here.

Reply:
What was said above - I think this is fine.

Reply:
Considering you are paying TC's for it and requires game knowledge to prep, its pretty fair.